scripts/stubtest/allowlist.txt

```diff
@@ -394,3 +394,6 @@ django.urls.resolvers.URLPattern.lookup_str
 django.urls.resolvers.URLResolver.url_patterns
 django.urls.resolvers.URLResolver.urlconf_module
 django.utils.connection.BaseConnectionHandler.settings
+
+# Potential mypy bug with metaclass
+django.contrib.gis.admin.options.GeoModelAdminMixin.gis_widget
```

Comment:
This also might be a bug with our plugin. Can you please try with and without `ClassVar`?

Reply:
In both cases it fails